constants.py

```diff
@@ -75,3 +75,6 @@ class ImageAugmentationStrength(Enum):
 
 # Data filtering constants
 BBOX_MIN_SIZE = 900 # Filter out images smaller than 30x30, TODO tweak
+
+# Output filtering constants
+HM_TO_KP_THRESHOLD = 0.05
```

Comment:
Likely a reasonable threshold, hopefully the visualization tests paired with the kp skeletons can give us insight into tuning this.

Reply:
I suspect a higher threshold is needed. We don't want uncertain activations, right?

Reply:
I'm just doing some testing and will likely raise it until it drops active keypoints!

Reply:
solid approach :)